extensions/amp-analytics/0.1/test/test-variables.js

```diff
@@ -205,7 +205,7 @@ describes.fakeWin('amp-analytics.VariableService', {amp: true}, env => {
     });
 
     function check(input, output, opt_bindings) {
-      const macros = Object.assign(variables.getMacros(), opt_bindings);
+      const macros = Object.assign(variables.getMacros(doc), opt_bindings);
```

Comment:
hmmm.. don't like this but am fine with this.

Reply:
yeah any dummy element will work. Would you rather I create a fake analytics element here?

Reply:
I would prefer an element. or a comment saying that doc serves as a dummy element.

Reply:
Done.